xdem/biascorr.py

```diff
@@ -0,0 +1,113 @@
+"""Bias corrections for DEMs"""
+from __future__ import annotations
+
+from typing import Callable, Optional
+
+import numpy as np
+import rasterio as rio
+
+import xdem
+
+
+class DirectionalBias(xdem.coreg.Coreg):
+    """
+    For example for DEM along- or across-track bias correction.
+    """
+
+    def __init__(self, bias_func : Callable = xdem.robust_stats.robust_polynomial_fit):  # pylint: disable=super-init-not-called
```

Comment:
Callables need to be subscripted to signify their inputs and outputs, no? They need to take two ndarrays as inputs (plus kwargs) and will return an int and an ndarray, right? If so, it should be:
```python
Callable[[np.ndarray, np.ndarray], tuple[int, np.ndarray]]
```

Reply:
And if it accepts kwargs, there's no current great way of doing it. One has to do:
```python
Callable[..., tuple[int, np.ndarray]]
```
where the ellipsis signifies "any number of arguments"

Reply:
Thanks a lot for the details, added!